jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/LambdaInputPanel.java

```diff
@@ -184,7 +182,7 @@ public String getInputText() {
         return StringUtil.nullize(inputText.getText().trim(), true);
     }
 
-    private class LocalLambdaSampleEvent extends LambdaSampleEvent {
+    private static class LocalLambdaSampleEvent extends LambdaSampleEvent {
```

Comment:
could this be an `object` ?

Reply:
Nah, if anything data class, but this is still Java